### PR TITLE
Capture CRN in session and apply default role

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/auth/defra-id/get-scopes.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
+const { USER } = require('../scopes')
+
 const getScopes = (roles) => {
-  return roles.map(role => {
+  const defraIdRoles = roles.map(role => {
     const roleData = role.split(':')
     return roleData.length > 1 ? roleData[1] : roleData[0]
   })
+  const ffdRoles = [USER]
+  return [...defraIdRoles, ...ffdRoles]
```
Comment (johnwatson484) [inline app/auth/defra-id/get-scopes.js:9]: Adding this as currently our prototypes are configured to allow anyone in this default user role.

Until we've explored the roles more, will just append the default role to every user.

File: app/auth/defra-id/validate-token.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const { getScopes } = require('./get-scopes')
 
 const validateToken = (decoded, _request, _h) => {
-  return { isValid: true, credentials: { scope: getScopes(decoded.roles), name: `${decoded.firstName} ${decoded.lastName}` } }
+  return { isValid: true, credentials: { scope: getScopes(decoded.roles), name: `${decoded.firstName} ${decoded.lastName}`, crn: decoded.contactId } }
```
Comment (johnwatson484) [inline app/auth/defra-id/validate-token.js:4]: Adding the CRN to the `credentials` property will mean it's available in our route handlers without needing to decode the cookie.

Similar to what we've done with `name`.

File: app/auth/get-redirect-path.js [hunk 3]
```diff
@@ -3,9 +3,9 @@ const getRedirectPath = (accessToken, refreshToken, redirect) => {
     redirect = '/landing-page/home'
   }
 
-  const tokenParams = `token=${accessToken}&refreshToken=${refreshToken}`
+  // TODO: refactor as part of access code spike
```
Comment (johnwatson484) [inline app/auth/get-redirect-path.js:6]: Removed the access token from the query string.  We know it's not secure and are going to ditch it.

Likely this function is redundant or needs changing, but can pick that up in the next ticket which is to solve that problem.  So have just simplified it for now.

**Review by rtasalem — APPROVED** [review]
LGTM

**Pull request merged**